backends/ref/ceed-ref.h

```diff
@@ -53,9 +53,7 @@ typedef struct {
 
 typedef struct {
   bool is_identity_qf, is_identity_restr_op;
-  CeedVector
-  *e_vecs;   /* E-vectors needed to apply operator (input followed by outputs) */
-  CeedScalar **e_data;
+  CeedVector *e_vecs;      /* All E-vectors, inputs followed by outputs */
   uint64_t *input_state;   /* State counter of inputs */
   CeedVector *e_vecs_in;   /* Input E-vectors needed to apply operator */
   CeedVector *e_vecs_out;  /* Output E-vectors needed to apply operator */
```

Comment:
Wait, why do we need `e_vecs` AND `e_vecs_in`/`e_vecs_out`?

Reply:
Good point - more legacy consequences. Let me see if it removes nicely.

Reply:
Actually... both do need to be there. One is the *full* evec and the other is the partial evec for the current block of elements.

Reply:
Ah, yes, thank you.  The new names and comments are much more clear.  (I'm so used to thinking about the GPU backends that I forgot this distinction existed in the CPU backends.)

Reply:
Cool. I'll start on the CUDA/HIP version of these changes tomorrow.